hepcrawl/spiders/desy_spider.py

```diff
@@ -100,27 +90,22 @@ def __init__(
         self.s3_input_bucket = s3_input_bucket
         self.s3_output_bucket = s3_output_bucket
 
-        self.s3_enabled = False if self.source_folder else True
-
-        if self.s3_enabled:
-            if not (
-                    self.s3_server
-                    and self.s3_key
-                    and self.s3_secret
-                    and self.s3_input_bucket
-                    and self.s3_output_bucket
-            ):
-                raise Exception("Missing s3 connection parameters")
-            else:
-                self.s3_connections = self.connect_to_s3()
-                self.s3_resource = self.get_s3_resource()
+        if not (
+                self.s3_server
+                and self.s3_key
+                and self.s3_secret
+                and self.s3_input_bucket
+                and self.s3_output_bucket
+        ):
+            raise Exception("Missing s3 connection parameters")
+
+        self.s3_connections = self.connect_to_s3()
+        self.s3_resource = self.get_s3_resource()
 
         if not os.path.exists(self.destination_folder):
             os.makedirs(self.destination_folder)
```

Comment:
is this still used?

Reply:
Yes. This is different s3 stuff :) We have 1 for DESY itself to get data, and there is second one in Pipeline (managed by scrapy) to upload documents. They are completely independent (unfortunately).

Reply:
O, you are asking about destination folder.... Sorry I looked automatically on "green" stuff ;)
Yeah, this is not necessary. :+1: